packages/relay-compiler/bin/RelayCompilerMain.js

```diff
@@ -157,6 +158,39 @@ function getLanguagePlugin(
   }
 }
 
+function getPersistQueryFunction(
+  config: Config,
+): ?(text: string) => Promise<string> {
+  if (config.persistFunction == null) {
+    return null;
+  } else if (typeof config.persistFunction === 'string') {
+    try {
+      // eslint-disable-next-line no-eval
+      const persistFunction = eval('require')(
+        path.resolve(process.cwd(), String(config.persistFunction)),
```

Comment:
String wrapper here to appease flow. Not sure why it's flagging since it's wrapped in a typeof check.

Reply:
Flow novice here, but perhaps Flow would only consider that when capturing the value in a local variable?

```js
const { persistFunction } = config
if (persistFunction == null) {
  return null;
} else if (typeof persistFunction === 'string') {
  // ...
}
```

Reply:
yeah, i suspect assigning to a local const would solve this